client/foundation/foundation_file_management_service.go

```diff
@@ -0,0 +1,41 @@
+package foundation
+
+import (
+	"context"
+	"net/http"
+
+	"github.com/terraform-providers/terraform-provider-nutanix/client"
+)
+
+type FileManagementService interface {
+	ListNOSPackages() (*ListNOSPackagesResponse, error)
+	ListHypervisorISOs() (*ListHypervisorISOsResponse, error)
+}
+
+type FileManagementOperations struct {
+	client *client.Client
+}
+
+//Lists the available AOS packages in Foundation
+func (fileManagementOperations FileManagementOperations) ListNOSPackages() (*ListNOSPackagesResponse, error) {
```

Comment:
```suggestion
func (fmo FileManagementOperations) ListNOSPackages() (*ListNOSPackagesResponse, error) {
```
Use smaller variable names as we already have type defined.

Reply:
Make changes everywhere.